src/react/components/common/keyboardManager/keyboardRegistrationManager.ts

```diff
@@ -1,6 +1,6 @@
 import Guard from "../../../../common/guard";
 import { KeyEventType } from "./keyboardManager";
-
+// FIXME: should KeyEventType enum in keyboardManager be moved here to avoid circular dependency?
```

Comment:
Are you getting warnings / errors about a circular dependency?

Reply:
no, it's not technically circular, but these two files import from each other unnecessarily. The should import in one direction. I can remove the comment or just move the enum from one file to another.

Reply:
I can also just write it up as a separate "style" bug.

Reply:
You're going to have the same case either way... Both files use the enum, but it is only ever defined in 1 of them.  Let's remove the comment and keep as is.